common.blocks/popup/popup.tests/gemini.blocks/test-nested/test.styl

```diff
@@ -1 +0,0 @@
-../../simple.blocks/test/test.styl
```

Comment:
Probably because of borschik (as post-processor) work will happen after pre-processor?